src/containers/ListingPage/components/Listing.jsx

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2016-present, NDLA.
+ *
+ *  This source code is licensed under the GPLv3 license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+import React, { PropTypes } from 'react';
+
+import { injectT } from '../../../i18n';
+import { findCategoryLabel, printSubjects } from '../../../util/listingHelpers';
+
+const Listing = ({ listings }) => {
+  const listItems = listings.filter(() => listings.length > 0).map(cover =>
+    <div className="produkt-container">
+      <div className="verktoy-bilde-div">
+        <img className="verktoy-img" alt={cover.coverPhoto} src={cover.coverPhoto} />
+      </div>
+      <div className="inner">
+        <a className="h2-tittel-lenke" href={cover.articleApiId}>
+          {cover.title.substr(0, 16)}
+        </a>
+        <div className="type-txt">{findCategoryLabel(cover.labels)}</div>
+        <p>{cover.description}</p>
+        <a href={`/article/${cover.articleApiId}`}>Les mer...</a>
+        <div>
+          {printSubjects(cover.labels).map(subject => <div><a className="tag-btn w-button" key={subject} href={`/listing/${subject}`}>{subject}</a></div>)}
+        </div>
+      </div>
+    </div>,
+            );
+
+  return (
+    <div className="main-content">
+      <div className="emneomrade-row">{listItems}</div>
+    </div>
+  );
+};
+
+
+Listing.propTypes = {
```

Comment:
Bra bruk av PropTypes! Det krever en del typing, men er verdt for dokumentasjon og warnings i console ved feil bruk.

Det er mulig å gjenbruke shapes. Se
Definisjoner: https://github.com/NDLANO/ndla-frontend/blob/master/src/shapes.js#L34
Bruk: https://github.com/NDLANO/ndla-frontend/blob/master/src/containers/TopicPage/TopicPage.jsx#L87